pkg/helm/utils.go

```diff
@@ -16,6 +17,11 @@ const (
 	REPO_URL     = "https://helm.tensorleap.ai"
 )
 
+func ValidateHelmInstallation(ctx context.Context) error {
+	// todo:
```

Comment:
Dont need to implement?
If not remove, if yes then add

Reply:
Sure, Implemented